src/components/dictionaryConcepts/components/MapType.jsx

```diff
@@ -33,12 +34,14 @@ mapType.propTypes = {
   updateEventListener: PropTypes.func,
   map_types: PropTypes.array,
   url: PropTypes.string,
+  source: PropTypes.string,
 };
 
 mapType.defaultProps = {
   url: '',
   map_type: '',
   index: 0,
+  source: 'CIEL',
```

Comment:
Don't we already have a constant for that?

Reply:
I have fixed it.